app/assets/stylesheets/welcome.scss

```diff
@@ -1,16 +1,27 @@
+
+
+#box {
+  z-index: 200;
+  height: 300px;
+  margin: 100px 0 -100px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `100px 0 -100px` instead of `100px 0 -100px 0`